openpower/package/sb-signing-framework/Config.in

```diff
@@ -0,0 +1,5 @@
+config BR2_PACKAGE_HOST_SB_SIGNING_FRAMEWORK
+        bool "OpenPOWER secureboot signing server interface"
+        default y if (BR2_OPENPOWER_PLATFORM && BR2_OPENPOWER_SECUREBOOT_ENABLED && (BR2_OPENPOWER_SECUREBOOT_SIGN_MODE = "production"))
```

Comment:
Remove dependency on BR2_OPENPOWER_SECUREBOOT_ENABLED to auto default the package as y